src/setup/setup.ts

```diff
@@ -0,0 +1,14 @@
+import { LIB_ID } from '../models/constants';
+import { Host } from '../models/host';
+import { Coordinator } from './coordinator';
+
+export function setupPostQuecast(coordinatorHost: Host = window): void {
+  if (!!(coordinatorHost as any)[LIB_ID]) {
```

Comment:
I'd allow the user to specify `LIB_ID` but that is a feature request, not sth to change in this ticket.

Reply:
LIB_ID exists solely to distinguish our lib from the other messages that may take place using postMessage. Why one would want to change it?